### ISSUE TITLE
update from 1.0.4 - problem with menu manager

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#3140]
I just tried update from version 1.0.4 to 2.1.7.  No visual problems.  

But when I tried to use menu manager (to remove bad looking menu), it doesn't work. I generated theme.xml but it doesn't help.  

![image](https://user-images.githubusercontent.com/5429548/40426729-17eff5b4-5e9c-11e8-855b-6d346e2eba15.png)

Frontend looks correct.  I can't find the link how to update old theme.

**Comment by CaMer0n** [comment]
@Jimmi08 I take it this is a custom theme? Are you able to send it to me for testing?

**Comment by Jimmi08** [comment]
Next week sure. But only if you want to find problem. It was my first bootstrap theme for e107 and I didnt know what I was doing most of time.  It just worked. I will use probably new theme for this site anyway.

**Comment by Jimmi08** [comment]
The same result with bootstrap3 theme:
![image](https://user-images.githubusercontent.com/5429548/41872675-eb84afa6-78c2-11e8-9074-fedf18d88b45.png)

**Comment by Jimmi08** [comment]
Source code: 
 
![image](https://user-images.githubusercontent.com/5429548/41873116-565cc2f4-78c4-11e8-8ed3-e8f5dcb8a650.png)

**Comment by CaMer0n** [comment]
@Jimmi08 Is this still an issue? If so, there must be a PHP somewhere.

**Comment by Jimmi08** [comment]
yes, it is.

**Comment by Jimmi08** [comment]
I could close this because I found other way how to update.

But there is something wrong with menus.  It's not by code, because there is no left of old code. 
It's uses Fizi theme.

At first sight Menu manager works.  But I couldn't find any news menu.   
When I check database, there were records with empty menu name, I deleted everything looking weird.


Still no menus from news in menu list.  I uses rescan plugin directory too.
